client/web/src/site-admin/SiteAdminRepositoriesPage.tsx

```diff
@@ -285,6 +331,9 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
 
     const licenseInfo = window.context.licenseInfo
 
+    const combinedError = error || extSvcError
+    const combinedLoading = loading || extSvcLoading
```

Comment:
should we rename error and loading to something more descriptive like:?
```suggestion
    const error = repoListError || extSvcError
    const loading = repoListLoading || extSvcLoading
```